xcube_clms/constants.py

```diff
@@ -18,5 +18,114 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+import logging
+
+CLMS_API_URL = "https://land.copernicus.eu/api"
+
+# Configuration constants
+TIME_TO_EXPIRE = 24  # (in hours)
+RETRY_TIMEOUT = 60  # (in seconds)
 
 DATA_STORE_ID = "clms"
+
+PRELOAD_CACHE_FOLDER = "preload_cache/"
+
+# CLMS API Constants
+SEARCH_ENDPOINT = "@search"
+DOWNLOAD_ENDPOINT = "@datarequest_post"
+TASK_STATUS_ENDPOINT = "@datarequest_search"
+CANCEL_ENDPOINT = "@datarequest_delete"
+
+BATCH = "batching"
+NEXT = "next"
```

Comment:
Such kind of context-free constant definitions do not make much sense.
Declare locally where used or not use at all or add a context prefix.

Reply:
cc44bdb17